lib/event-service.js

```diff
@@ -0,0 +1,44 @@
+var EVENTSERVICE_SERVICE_UUID          = 'E95D93AF251D470AA062FA1922DFA9A8'.toLowerCase();
+var MICROBITEVENT_CHARACTERISTIC_UUID = 'E95D9775251D470AA062FA1922DFA9A8'.toLowerCase();
+
+var CLIENTREQUIREMENTS_CHARACTERISTIC_UUID = 'E95D23C4251D470AA062FA1922DFA9A8'.toLowerCase();
+var CLIENTEVENT_CHARACTERISTIC_UUID = 'E95D5404251D470AA062FA1922DFA9A8'.toLowerCase();
+
+var EventService = function() {
+};
+
+EventService.prototype.hasEventService = function() {
+  return this.hasService(EVENTSERVICE_SERVICE_UUID);
+};
+
+EventService.prototype.onMicrobitEventChange = function(data) {
+  if (data.length !== 4) {
+    return;
+  }
+  var event = data.readInt16LE(0);
+  var value = data.readInt16LE(2);
+  this.emit('microbitEventChange', event, value);
+};
+
+EventService.prototype.subscribeMicrobitEvents = function(client_reqs, callback) {
+  // specifying which events we want to be notified about
+  this.writeClientEventRequirements(client_reqs);
+

```

Comment:
Would a user call this more than once? Like once per event type?

If so, we should make sure the subscribe below is only called on the first call.